test/karma/helpers/ParentClickView.js

```diff
@@ -0,0 +1,35 @@
+var _ = require('underscore');
+var TorsoView = require('./../../../modules/View');
+var ChildClickView = require('./ChildClickView');
+
+module.exports = TorsoView.extend({
+  events: {
+    'click div.parent' : 'myClick'
+  },
+  initialize: function() {
+    this.spyHook();
+
+    spyOn(this, 'myClick');
+    spyOn(this, 'afterMyEvent');
+    spyOn(this, 'afterMyDeactivatableEvent');
```

Comment:
That's not really fixing it - rather than to separate test source objects and test framework code, that's just further mixing it by naming the test source object by a seemingly insignificant test framework attachment.

Any reason you don't want to do one of the two options I suggested?